R/csv-utils.R

```diff
@@ -1,19 +1,42 @@
-#' @keywords internal
-mrgda_read_csv <- function(.file) {
+#' Read csv file
+#'
+#' @description
+#' Reads csv files and formats the data to replace any "." with
+#' NA values.
+#'
+#' @param .file file path to csv file
+#'
+#' @export
+read_nm_csv <- function(.file) {
 
-  data.table::fread(
-    file = .file,
-    sep = ",",
-    quote = FALSE,
-    na.strings = "."
-  ) %>%
+  df <-
+    data.table::fread(
+      file = .file,
+      sep = ",",
+      quote = FALSE,
+      na.strings = "."
+    ) %>%
     as.data.frame() %>%
     suppressMessages()
 
+  cli::cli_alert_success(paste0("Successfully read: '", .file, "'"))
```

Comment:
It would be nice if these functions didn't send alerts / messages, especially when every time you call the function you get the (same) alert.

Reply:
@kylebaron Agreed, just removed the `cli` components to the functions